web_services/foxx/brightid/index.js

```diff
@@ -319,6 +345,7 @@ router.get('/operations/:hash', handlers.operationGet)
 router.get('/verifications/:context/:contextId', handlers.verificationGet)
   .pathParam('context', joi.string().required().description('the context in which the user is verified'))
   .pathParam('contextId', joi.string().required().description('the contextId of user within the context'))
+  .queryParam('verification', joi.string().description('request a customized verification.'))
```

Comment:
the verification expression that will be evaluated for user

Reply:
It's done
https://github.com/BrightID/BrightID-Node/commit/2dc63217596958c40202d68ef012dc53a483d19e